randovania/game_connection/connector/remote_connector.py

```diff
@@ -11,13 +12,12 @@
 from randovania.lib import enum_lib
 
 if typing.TYPE_CHECKING:
-    import uuid
-
     from randovania.game.game_enum import RandovaniaGame
     from randovania.game_description.db.area import Area
     from randovania.game_description.db.region import Region
 
-PickupEntryWithOwner = tuple[str, PickupEntry]
+# provider_name, PickupEntry, PickupIndex, provider_uuid
+RemotePickupTuple = tuple[str, PickupEntry, PickupIndex, uuid.UUID]
```

Comment:
at this point we should just make it a class inheriting from NamedTuple.

And somewhere in network common so that it can also be used by multiplayer_session.py